### ISSUE TITLE
Proxy support and settings

### ISSUE THREAD

**Issue by Reportonly** [tjg1/nori#110]
Possibly separate for different domains

**Comment by tjg1** [comment]
Nori should already respect system-wide proxy settings.

I'm wondering why you would need granular control for each API? Is it to bypass filtering?

**Comment by Reportonly** [comment]
Yes, it is for bypassing filtering so i dont want use system proxy. There is absolutely no need in separate settings for different domains.
